_component_design/buttons-simple.md

```diff
@@ -61,6 +61,26 @@ Consider the following best practices when you construct a button:
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
+## Simple button types
+
+Simple buttons types include *primary*, *secondary*, and *tertiary* buttons.
+
+{% figure [caption:"Simple button types"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/simple-buttons/buttons-simple-hero.png" width="569"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
 ## Button labels
```

Comment:
button labels should go below the specifications, shouldn't it?

Reply:
@palindromebart, this is all set now. Please LGTM! Woot!